src/compound_targets.py

```diff
@@ -8,6 +8,10 @@
     "cursorless_range_connective",
     desc="A range joiner that indicates whether to include or exclude anchor and active",
 )
+mod.list(
+    "cursorless_range_column",
```

Comment:
Maybe `cursorless_range_type`, `cursorless_range_modifier`, or `cursorless_range_axis`?  Not sure if we'll ever generalize this one, so maybe what you have is fine 🤷‍♂️

Reply:
range type could work